docs/network/architecture.md

```diff
@@ -13,16 +15,21 @@ These components are instances of binaries in the [Octez software suite](https:/
 ## Sequencer
 
 Etherlink relies on a sequencer to publish transactions.
-It receives transactions from EVM nodes, puts them in order, and packages them into a blueprint, which is a group of Etherlink transactions.
+The sequencer receives transactions from EVM nodes, puts them in order, and packages them into an Etherlink block.
 
-Each blueprint contains:
+The sequencer generates blocks at a variable rate, depending on demand.
+Currently, it generates a block at least every 6 seconds, even if the block is empty.
+As demand increases, it generates blocks more quickly, up to a block every 500ms.
+These values can change with Etherlink upgrades.
```

Comment:
here it's not etherlink kernel upgrade, but the sequencer node upgrade, or even a config to change.
I'm not sure we should state that it can change here ?
Is upgrade only used for upgrade voted by users, or do we introduce node upgrade in the documentation too ?

Reply:
I'll merge this PR for now and make a note to cover the Etherlink upgrade process. I don't know how much detail we'll want to go into.